### PR TITLE
Update Component design to Match MockUp

### PR DESCRIPTION
**Pull request by nobioma1**

### CODE REVIEW

File: src/components/Slack/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const parsed = qs.parse(search);
+  const { '?code': code } = parsed;
+
+  useEffect(() => {
+    getSlackToken(code);
+  }, [getSlackToken, code]);
+
+  return (
+    <React.Fragment>
+      <SideNav />
```
Comment (maximesalomon) [inline src/components/Slack/index.js]: You can use <> and </> for React.Fragment

Comment (nobioma1) [reply src/components/Slack/index.js]: Okay. Thanks for the fix on the React.Fragment. Working with @samarv on that to fix it on the netlify ENV variables

Comment (nobioma1) [reply src/components/Slack/index.js]: @maximesalomon It has been fixed

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+[build]
+  command = "CI=true npm run test && npm run build"
+  publish = "build"
+
+# Deploy Preview context: All deploys generated from a pull/merge request
+# will inherit these settings.
+[context.deploy-preview.environment]
+  REACT_APP_CLIENT_ID = "711954789234.738124914208"
+# Not so secret after all 
+  REACT_APP_CLIENT_SECRET = "a6b2dcbfeff77561afde34eea60908cd"
```
Comment (maximesalomon) [inline netlify.toml]: Shouldn't we use an env variable here instead of our API secret?

Comment (samarv) [reply netlify.toml]: We tried saving it in env variables but netlify.toml file can't read from netlify's environment variable. 

Any how, these are just for the staging environment. 
We are storing production variable in env.

**Pull request merged**